test/pass.jsx

```diff
@@ -16,3 +16,8 @@ const DestructuringComponent = props => (
 DestructuringComponent.propTypes = {
   message: PropTypes.string.isRequired,
 };
+
+// react/jsx-props-no-spreading
+const SpreadProps = props => <div role="banner" {...props} />;
```

Comment:
Can we add more tests for all rules we've disabled?

Reply:
Sounds reasonable, will add some more 😄

Reply:
Tests added